doc/minutes/2022-12-02.md

```diff
@@ -0,0 +1,98 @@
+---
+title:  "Second meeting to organize YES award"
+date:   "02/12/2022"
+output: github_document
+---
+
+**Attendees:** Lea Francia, Roberto Mediavilla, Daniel Morillo, Cristina
+Rodríguez, Jesús Sierralaya
+
+**Scribe:** Cristina Rodríguez
+
+**Schedule:** \<11:00\> - \<11:31\> <!-- # Init time - End time -->
+
+# Action points
+
+## Approval of the previous minutes
+
+We left pending for the next meeting to approve the minutes of the first meeting
+taken by Jesus due to github usability issues.
+
+## Use of 'sourcetree' for manipulating branches on Git
+
+We discuss how version history is recorded and who is changing what. A good tool
+to manage branches is 'sourcetree', which can be easily downloaded from
+[here](https://www.sourcetreeapp.com/).
+
+## Use of canonical mode
+
+The use of the canonical mode saves us conflicts when working with people who
+prefer to work with different modes (Source or Visual). This is because it
+establishes conventions that are always applied in Visual to Source mode. In our
+project it is set up to get as few headaches as possible in this matter.
+
+## Skeleton of the proposal
+
+After discussing different usability problems with Git and solving the issue of
+the branches to work on, Roberto presents us how he has structured the skeleton
+of the proposal. In this sense, it has been based on three key concepts:
+
+-   data management
+-   [literate
+    computing](https://medium.com/@mryap/what-is-literate-computing-6b3de57595a6)
+-   version control It is decided that each of the four will review a part of
+    the skeleton and elaborate it in maximum 400 characters, and that Roberto
+    will be the one to finally review it as a whole.
+
+To work in each of these sections, each one of us will create a branch that
+comes out of the branch `template-fillin` and that corresponds to our section.
+For example, in the case of Cris it would be `template-fillin/reusability`
+
+#### Tasks
+
+-   \Lea Francia\>:
+
+    Outputs: What has the initiative achieved so far?
```

Comment:
State the task as a verb to make it clearer: e.g. `Fill in field "Outputs: What has the initiative achieved so far?"`

Also, do not separate member and task (write where the wildcard is in the template)